components/wpt-results.html

```diff
@@ -146,40 +153,31 @@
         testFileResults.forEach(result => {
```

Comment:
What is one "result"? Is it all of the results for one run (on one browser, on one platform)? Does it correspond to one "results.json" file or something like that?